package.json

```diff
@@ -30,7 +30,14 @@
     "date-fns": "^2.28.0",
     "fn-args": "^5.0.0",
     "fs-extra": "^10.0.1",
-    "lodash": "^4.17.21",
+    "i": "^0.3.7",
```

Comment:
what is this strange package? I don't see it used anywhere

Reply:
Great observation. Let me correct this